components/Card.js

```diff
@@ -1,12 +1,17 @@
 import propTypes from 'prop-types'
 import ViewMoreLessButton from '../components/ViewMoreLessButton'
-import { useState } from 'react'
+import BenefitTasks from './BenefitTasks'
+import { useState, useEffect } from 'react'
```

Comment:
We can remove `useEffect` since we are not using it :) (Sorry I forgot to remove it)

Reply:
done